service-catalog/components/service-catalog-postgres-cluster.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: service-catalog-pg-cluster
```

Comment:
I think this name and the file name should be the same, it makes it easier when reviewing and keeping track of what resources are referenced where

Reply:
good catch, will fix that :slightly_smiling_face: